package.json

```diff
@@ -50,6 +50,7 @@
     "appium-ios-driver": "4.x",
     "appium-mac-driver": "1.x",
     "appium-mac2-driver": "^0.x",
+    "appium-roku-driver": "^1.0.0",
```

Comment:
does this work? I observe the current driver version is 0.1.1
Perhaps we need `^0.x` instead

Reply:
Fixed the version.
Thanks very much.